v2.0/information-schema.md

```diff
@@ -168,8 +189,8 @@ The `views` view identifies [views](views.html) in the cluster.
 
 Column | Description
 -------|-----------
-`TABLE_CATALOG` | Always equal to `def` (CockroachDB does not support the notion of catalogs).
-`TABLE_SCHEMA` | Name of the database that the view reads from.
+`TABLE_CATALOG` | Name of the database that the view reads from.
```

Comment:
Both the original doc text and the new one are incorrect. the `table_xxx` columns in this table refer to the view itself, not the table(s) being read from. A view can read from multiple tables.

Reply:
Fixed.